be/src/runtime/load_channel_mgr.cpp

```diff
@@ -126,6 +110,15 @@ Status LoadChannelMgr::open(const PTabletWriterOpenRequest& params) {
     return Status::OK();
 }
 
+void LoadChannelMgr::_pending_if_hard_limit_exceeded() {
+    std::unique_lock<std::mutex> l(_lock);
+    while (_hard_limit_reached) {
+        LOG(INFO) << "Reached the load channel manager mem limit " << _mem_tracker->limit()
```

Comment:
`LOG(INFO) ` may print frequently, consider => `VLOG_NOTICE`

Reply:
It's at most as frequent as the `reducing memory usage...` log, I think it's acceptable?